python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/abstract_interp_lib_gen.py

```diff
@@ -2232,6 +2232,36 @@ def aten〇native_batch_norm〡dtype(input_rank_dtype: Tuple[int, int], weight_r
     assert is_float_dtype(input_dtype)
     return input_dtype, input_dtype, input_dtype
 
+@check_dtype_function([
+    Invocation(TensorOfShape(2, 3, 4, dtype=torch.float32), dim=0, half_to_float=False)])
```

Comment:
Can you use the helper functions for testing? These create invocations that make very thorough checks of the dtype functions. Same comment applies to the rest of the functions. See:

https://github.com/llvm/torch-mlir/blob/d3a49fd38f228d635cce66e3d2f0170c4023e686/python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/abstract_interp_lib_gen.py#L1305-L1312